internal/resources-validation.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+jobs:
+# Spin up a Linux VM for validating code.
+- job: png_resources_validation
+  displayName: PNG Resources Validation
+  pool:
+    vmImage: 'ubuntu-latest'
+
+  steps:
+    - script: |
+        sudo apt update
+        sudo apt install -y pngcrush
+      displayName: 'Install Dependencies'
```

Comment:
Is it possible to have the displayName key the first item in the list. It make easier to read the code (code as documentation).

```yaml
  steps:
    - displayName: 'Install Dependencies'
      script: |
        sudo apt update
        sudo apt install -y pngcrush

    - displayName: 'Process PNG files using pngcrush'
      script: |
        # Print the full paths of PNG files
        find . -type f -name "*.png" -print
```

Reply:
Tried that, but the CI failed. It seems that the displayName property needs to be placed after the task type, such as script, bash, powershell, etc. and the task type needs to be the first property of each step.